app/frontend/src/components/Scroll/_ChevronScroll.scss

```diff
@@ -0,0 +1,62 @@
+@use "../Basics/_colors" as *;
+@use "components/Basics" as *;
+
+.chevron-scroll-outer-container {
+  white-space: nowrap;
+  display: flex;
+  width: 1088px;
+  height: 47px;
+}
+
+.chevron-scroll-child-container {
+  scroll-snap-type: x mandatory;
+  overflow: auto;
+  display: flex;
+  align-items: center;
+  gap: 16px;
+}
+
+.chevron-scroll-child-container::-webkit-scrollbar {
+  display: none;
+}
+
+.chevron-scroll-left-btn {
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
I could be mistaken, since it has been a while since I looked at our existing classes, but think these can be `flex-align-center justify-center` with no loss in functionality.

Reply:
ya good call :+1: